assets/css/custom.css

```diff
@@ -424,7 +424,7 @@ body:last-child .btn-u, x:-moz-any-link {
 .browse-courses {
     display: inline-block;
     border-radius: 6px;
```

Comment:
Border radius is used in 8 different places. We can do it more DRY if we define a single style (e.g. ".rounded") and add it to all elements that require it. If we want to change the rounding value in the future, we'll do it in one single place.